packages/mobile/src/screens/profile-screen/ProfileHeader/Bio.tsx

```diff
@@ -11,7 +11,8 @@ const MAX_BIO_LINES = 2
 
 const useStyles = makeStyles(({ typography, palette, spacing }) => ({
   root: {
-    marginBottom: spacing(2)
+    marginBottom: spacing(2),
+    lineHeight: typography.fontSize.small * 1.5
```

Comment:
Just double checking cuz we usually do 1.3!

Reply:
Good callout, spec has 1.5! One of the biggest differences between this and spec tbh